typescript-selenium-webdriver-sample/jest.config.js

```diff
@@ -13,6 +13,12 @@ module.exports = {
     // See https://github.com/facebook/jest/issues/2713 for details.
     testRunner: 'jest-circus/runner',
 
+    moduleNameMapper: {
```

Comment:
This is a pretty unpleasant hack to require; I'd kind of expect us to want to fix this before we start adding the new reporter to sample code. At the very least I'd want a comment referencing a bug number

Reply:
I agree. I don't know any way around this.

Reply:
It's documented: https://github.com/OfficeDev/office-ui-fabric-react/wiki/Fabric-6-Release-Notes#lib-commonjs

I will see if I can find a workaround. I don't like that anyone building tests using `jest` would have to do this.